spec/autocomplete-manager-async-spec.coffee

```diff
@@ -4,6 +4,9 @@ describe 'Async providers', ->
 
   beforeEach ->
     runs ->
+      jasmine.unspy(window, 'setTimeout')
+      jasmine.unspy(window, 'clearTimeout')
```

Comment:
Because atom breaks setTimeout in tests: https://github.com/atom/atom/blob/964809373bbf6e621a1b307c4fae372b8fe0e525/spec/spec-helper.coffee#L91-L92